test/utils/application.data.spec.ts

```diff
@@ -473,9 +473,9 @@ describe("Application data utils", () => {
       MANAGING_OFFICER_CORPORATE_OBJECT_MOCK
     ]
   ])(`allManagingOfficers with %s returns expected array`, (_, moKey, moMock) => {
-    const allMOs = allManagingOfficers({ [UpdateKey]: { [moKey]: [moMock] } });
+    const allBOs = allManagingOfficers({ [UpdateKey]: { [moKey]: [moMock] } });
 
-    expect(allMOs).toEqual([moMock]);
+    expect(allBOs).toEqual([moMock]);
```

Comment:
think you maybe need to pull the latest from main for this file, I had these called BOs but changed it to MOs

Reply:
Fixed